packages/devextreme/testing/helpers/diagramHelpers.js

```diff
@@ -80,7 +81,7 @@ export function findPropertiesPanelToolbarItem($diagramElement, label) {
         });
 }
 export function getToolbarIcon($button) {
-    return $button.find('.dx-dropdowneditor-field-template-wrapper').find('.dx-diagram-i, .dx-icon');
+    return $button.find(`.${DROP_DOWN_EDITOR_BEFORE_FIELD_ADDON}`).find('.dx-diagram-i, .dx-icon');
```

Comment:
[minor]
Will it be difficult for you to import class names? Or create variables?